recipe/meta.yaml

```
@@ -27,6 +25,7 @@ requirements:
   run:
     - python
     - types-pytz >=2022.1.1
+    - {{ pin_compatible('numpy', lower_bound='1.26.0')}}
```

```suggestion
    - numpy >=1.26 # [py<313]
```
pin_compatible is to ensure consistency with the version used in host. When only the python numpy api is used, we don't need to set it in host and therefore don't need pin_compatible.

---

That's what I get for trying to be fancy!